src/indexmap.rs

```diff
@@ -0,0 +1,2539 @@
+#![allow(unsafe_code)]
```

Comment:
Module level docs please.

Reply:
Would module level docs be explaining that this code is copied from heapless and not to touch it?

Reply:
> Would module level docs be explaining that this code is copied from heapless and not to touch it?

Yep, although I would suggest that future contributors use caution, rather than not touch it at all.